app/views/chomp_sessions/show.html.erb

```diff
@@ -2,10 +2,12 @@
   <div class="row justify-content-center">
     <div class="col-12 col-lg-8">
       <div class="form-login">
-        <h1 class="mb-3">Input Preferences</h1>
+        <h1 class="mb-3">Input Preferences</h1> <%= render "timer" %>
```

Comment:
can move the <render timer> into the next line?

Reply:
render moved to next line in latest push